packages/core/src/components/SortableTable/SortableTable.tsx

```diff
@@ -188,49 +224,38 @@ const DataTableRow = pure(({ row, columns, handleRowClick, style }) => {
  *     ev.preventDefault();}}/>)
  * }
  *
+ * // XXX (@koroeskohr): supposedly this is leftover from your internal doc
  * @deprecated use shared/components/DataGrid
  */
-class SortableTable extends React.Component {
-  static propTypes = {
-    // TODO: figure out how to make id of the object requried while others are dynamic
```

Comment:
I didn't integrate that in the type system yet, as I failed to find a proper typing for `data`